src/engine/Docs/Labels.md

```diff
@@ -8,7 +8,7 @@ to be drawn and updated on-screen.
 ```js
 var game = new ex.Engine();
 // constructor
-var label = new ex.Label('Hello World', 50, 50, '10px Arial');
+var label = new ex.Label('Hello World', 50, 50, 'Arial');
```

Comment:
Thanks! Here are more details about what I was referring to.

This file will be deleted in my other PR: https://github.com/excaliburjs/Excalibur/pull/1669

Therefore, any updates should be added to this new docs file:

https://github.com/excaliburjs/excaliburjs.github.io/blob/site/docs/14-ui.mdx

Reply:
@kamranayub, ah, thank you for pointing that out.

I was instead trying to find a documentation file named "label" (to mirror the `Label.md` changed here) in the other repository, but obviously none existed. I should've looked harder to find that the label documentation had been moved into `14-ui.mdx`.

Anyway, the corresponding pull request on the other website repository is: excaliburjs/excaliburjs.github.io#53